lib/project_types/script/layers/infrastructure/languages/task_runner.rb

```diff
@@ -5,15 +5,35 @@ module Layers
     module Infrastructure
       module Languages
         class TaskRunner
-          TASK_RUNNERS = {
-            "assemblyscript" => AssemblyScriptTaskRunner,
-            "typescript" => TypeScriptTaskRunner,
-            "wasm" => WasmTaskRunner,
-          }
-
           def self.for(ctx, language, script_name)
-            raise Errors::TaskRunnerNotFoundError unless TASK_RUNNERS[language]
-            TASK_RUNNERS[language].new(ctx, script_name)
+            task_runners = {
+              "assemblyscript" => AssemblyScriptTaskRunner,
+              "typescript" => TypeScriptTaskRunner,
+              "wasm" => WasmTaskRunner,
+            }
```

Comment:
I had to move this from the constant to part of the `.for` method because it was causing load order issues during tests. https://shopify.slack.com/archives/G01LAN7M5LP/p1643752926365439